bool request_pose
---
float64[7] pose_values